index.bs

```diff
@@ -235,9 +243,9 @@ The following steps are taken:
 1. If _stage_ is *undefined* then let _stage_ be "ready".
 1. Return the result of transforming Resolve(_loader_, _name_, _referrer_) with a fulfillment handler that, when called with argument _key_, runs the following steps:
   1. If _stage_ is "fetch", then:
-    1. Return RequestFetch(_loader_, _key_).
+    1. Return the result of transforming RequestFetch(_loader_, _key_) with a new pass-through promise.
   1. If _stage_ is "translate", then:
-    1. Return RequestTranslate(_loader_, _key_).
+    1. Return the result of transforming RequestTranslate(_loader_, _key_) with a new pass-through promise.
```

Comment:
exposing a wrapping promise instead of the internal slot returned by `RequestTranslate()`